semantic_search/main.py

```diff
@@ -115,4 +115,10 @@ async def query(query: Query) -> List[Dict[str, float]]:
     top_k_indicies = top_k_indicies.reshape(-1).tolist()
     top_k_scores = top_k_scores.reshape(-1).tolist()
 
+    if int(query.query.uid) in (top_k_indicies):
+        index = top_k_indicies.index(int(query.query.uid))
+        del top_k_indicies[index], top_k_scores[index]
+    else:
+        del top_k_indicies[-1], top_k_scores[-1]
```

Comment:
This might be the source of your -1 error. Not sure you can write `del` statements on the same line like this?

Reply:
It happens even if I structure it separately on different lines. I think it is something else.